pghoard/wal.py

```diff
@@ -42,107 +44,156 @@ class WalBlobLengthError(ValueError):
     """WAL blob is shorter than the WAL header"""
 
 
-WalHeader = namedtuple("WalHeader", ("version", "timeline", "lsn", "filename"))
+WalHeader = namedtuple("WalHeader", ("version", "lsn"))
 
 
-def read_header(blob):
-    if len(blob) < WAL_HEADER_LEN:
-        raise WalBlobLengthError(
-            "Need at least {} bytes of input to read WAL header, got {}".format(WAL_HEADER_LEN, len(blob))
+class LSN:
+
+    SEGSIZE = WAL_SEG_SIZE
```

Comment:
Looks strange: why replace one constant with another? Do you expect to overwrite it in tests?

Reply:
Maybe that's premature, but the idea is to be able to create an LSN subclass dynamically for the server if it has been compiled using a non default wal segment size. 

I can remove it for now.

Reply:
How often do you expect other wal seg sizes to come up?

Reply:
I removed it for now.